api/src/certification/session/infrastructure/repositories/session-repository.js

```diff
@@ -211,11 +212,12 @@ const hasNoStartedCertification = async function (sessionId) {
   return !result;
 };
 
-const countUncompletedCertifications = async function (sessionId) {
+const countUncompletedCertificationsAssessment = async function (sessionId) {
   const { count } = await knex
-    .count('id')
+    .count('certification-courses.id')
     .from('certification-courses')
-    .where({ sessionId, completedAt: null })
+    .join('assessments', 'certification-courses.id', 'certificationCourseId')
+    .where({ sessionId, state: CertificationAssessment.states.STARTED })
```

Comment:
Pour l'instant il n'y a que l'état started qui correspond à une certif non terminée. Pour être plus robuste, est-ce que l'on cherche les certifs non terminées ?
```suggestion
    .whereIn({ sessionId, state: [CertificationAssessment.states.COMPLETED,
CertificationAssessment.states.ENDED_BY_SUPERVISOR,
CertificationAssessment.states.ENDED_DUE_TO_FINALIZATION] })
```

Reply:
`whereNotIn` du coup pour ta suggestion non @Osirisxxl  ?

Reply:
Mais la tu propose l'inverse de ce que l'on veut!?

Reply:
Ah c'est edité!

Reply:
C'est pertinent mais ca impliquerait de faire le changement à d'autres endroit et je ne veux pas trop impacter le code. Ces histoires de status pourraient effectivement etre revus!